Reword sentence in first post (#709)
phil-opp/blog_os: blog/content/second-edition/posts/01-freestanding-rust-binary/index.md

[`Copy`]: https://doc.rust-lang.org/nightly/core/marker/trait.Copy.html
[copy code]: https://github.com/rust-lang/rust/blob/485397e49a02a3b7ff77c17e4a3f16c653925cb3/src/libcore/marker.rs#L296-L299

> Providing our own implementations of language items would be possible, but this should only be done as a last resort. The reason is that language items are highly unstable implementation details and not even type checked (so the compiler doesn't even check if a function has the right argument types). Fortunately, there is a more stable way to fix the above language item error.

Reviewer: Thanks for the pull request! I'm not a native speaker, so I might be wrong, but adding "our" here doesn't feel ideal either.

How about rewording the sentence a bit:

```suggestion
While providing custom implementations of language items is possible, it should only be done as a last resort. The reason is that language items are highly unstable implementation details and not even type checked (so the compiler doesn't even check if a function has the right argument types). Fortunately, there is a more stable way to fix the above language item error.
```

What do you think?
